extensions/panache/hibernate-orm-panache-kotlin/deployment/src/main/java/io/quarkus/hibernate/orm/panache/kotlin/deployment/KotlinPanacheResourceProcessor.java

```diff
@@ -63,6 +64,45 @@
     static final String OBJECT_SIGNATURE = toBinarySignature(Object.class);
     static final String CLASS_SIGNATURE = toBinarySignature(Class.class);
 
+    static final org.objectweb.asm.Type CLASS_TYPE = org.objectweb.asm.Type.getType(Class.class);
+    static final org.objectweb.asm.Type OBJECT_TYPE = org.objectweb.asm.Type.getType(Object.class);
+    static final List<org.objectweb.asm.Type> REPOSITORY_TYPES = asList(
+            org.objectweb.asm.Type.getType(PanacheRepositoryBase.class),
+            org.objectweb.asm.Type.getType(PanacheRepository.class));
+    static final List<org.objectweb.asm.Type> ENTITY_TYPES = asList(
+            org.objectweb.asm.Type.getType(PanacheEntityBase.class),
+            org.objectweb.asm.Type.getType(PanacheEntity.class),
+            org.objectweb.asm.Type.getType(PanacheCompanion.class));
+    public static final List<org.objectweb.asm.Type> PRIMITIVE_TYPES = asList(
```

Comment:
I wonder if we already have this somewhere else...

Reply:
I couldn't find it anywhere

Reply:
Yeah, me neither